doc/refactoring_design.md

```diff
@@ -0,0 +1,481 @@
+# Keeper Contract Design for Tethys
+
+***DISCLAIMER: THIS IS A WORK IN PROGRESS***
+
+This document provides technical details and describes the high-level design of keeper contract in ocean protocol.
+
+## 1. Overview
+
+### 1.1 Core Modules
+
+From the top-level point of view, there are total 9 moduels as shown in the below:
+
+<img src="img/refactoring/modules.jpg" width="1000" />
+
+* **Ocean Token**: the native ERC20 token in Ocean network
```

Comment:
Are we going to use [ERC721](http://erc721.org/) or [ERC20](https://theethereum.wiki/w/index.php/ERC20_Token_Standard) and why ?

Reply:
Ocean token will be ERC20. It is fungible tokens.